src/pybamm/models/full_battery_models/lithium_ion/electrode_soh.py

```diff
@@ -276,7 +276,7 @@ class ElectrodeSOHSolver:
     ----------
     parameter_values : :class:`pybamm.ParameterValues.Parameters`
         The parameters of the simulation
-    param : :class:`pybamm.LithiumIonParameters`, optional
+    self.param : :class:`pybamm.LithiumIonParameters`, optional
```

Comment:
The argument name does not change
```suggestion
    param : :class:`pybamm.LithiumIonParameters`, optional
```